packages/react-mutation-mapper/src/component/lollipopMutationPlot/LollipopMutationPlotControls.tsx

```diff
@@ -215,6 +220,15 @@ export default class LollipopMutationPlotControls extends React.Component<
         );
     }
 
+    protected get percentToggle() {
```

Comment:
this will have negligible benefit in this case, but it's good to make a getter that returns jsx @computed to avoid unnecessary recomputation.

Reply:
yea, this is mostly to group the toggle with the sliders on the left when the filter reset panel is triggered as opposed to the legend and download controls